### PR TITLE
Dutch translations for index.js

### PR DESCRIPTION
**Pull request by jelle619**

### CODE REVIEW

File: texts/index.js [hunk 1]
```diff
@@ -227,3 +227,41 @@ texts.push([
...
+	"Doe mee!", // 17
+	"Kijk wat we aan het doen zijn!", // 18
+	"De emulator kan nog geen <b>commerciële spellen</b> afspelen. <br>Er worden een paar homebrew-spellen ondersteund. Zie de <a href=\"compatibility.html?lang=en\">compatibiliteitslijst</a> voor verdere informatie!", // 19
+	"Homebrew Game Port", // 20
+	"Homebrew Game", // 21
+	"Homebrew Hulpmiddel", // 22
+	"Help ons beter worden!", // 23
+	"Je kunt op de volgende manieren bijdragen aan de emulator:", // 24
+	"Licht ons in over problemen of maak Pull Requests op onze <a href=\"https://github.com/Vita3K/Vita3K\">GitHub repository</a>.", // 25
+	"Update de <a href=\"https://github.com/Vita3K/homebrew-compatibility/issues\">homebrew compatibiliteitslijst</a>.", // 26
```
Comment (VelocityRa) [inline texts/index.js]: compatibiliteitslijst?

File: texts/index.js [hunk 2]
```diff
@@ -227,3 +227,41 @@ texts.push([
...
+	"Vita3K is ‘s werelds eerste experimentele open-source PlayStation Vita emulator voor Windows en MacOS.", // 10
+	"Wees ervan bewust dat het in staat stellen van illegale activiteiten niet de bedoeling van deze emulator is.", // 11
+	"Kies een passende versie!", // 12
+	"Binnenkort!", // 13
+	"Haal binnen via", // 14
+	"Neem contact met ons op!", // 15
+	"Neem contact op met de ontwikkelaars en andere gebruikers wanneer je maar wilt. Doe dit het liefst via Discord, maar de keuze ligt bij jou!", // 16
+	"Doe mee!", // 17
+	"Kijk wat we aan het doen zijn!", // 18
+	"De emulator kan nog geen <b>commerciële spellen</b> afspelen. <br>Er worden een paar homebrew-spellen ondersteund. Zie de <a href=\"compatibility.html?lang=en\">compatibiliteitslijst</a> voor verdere informatie!", // 19
```
Comment (VelocityRa) [inline texts/index.js]: lang code is incorrect, needs to be `lang=nl`

**Comment by VelocityRa** [comment]
Hello, you need to do compatibility.js too for a complete translation.

File: texts/index.js (hunk 1 above)
Comment (jelle619) [reply texts/index.js]: Yup.
One word.
No spaces.

Comment (VelocityRa) [reply texts/index.js]: lol

Comment (jelle619) [reply texts/index.js]: Well okay technically they’re still two words but it has to be written without a space inbetween.

**Comment by jelle619** [comment]
compatibility.js has also been translated now.

**Comment by jelle619** [comment]
Lang query has been hotfoxed.

**Pull request merged**